specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-05-01-preview/apimpolicyrestrictionsvalidation.json

```diff
@@ -0,0 +1,79 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ApiManagementClient",
+    "description": "REST API for performing policy restrictions on any created/updated policies in an Azure API Management deployment.",
+    "version": "2023-05-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/validatePolicies": {
+      "post": {
+        "tags": [
+          "PolicyRestrictionsValidations"
+        ],
+        "operationId": "PolicyRestrictionValidations_ListByService",
```

Comment:
>PolicyRestrictionValidations_ListByService [](http://example.com/codeflow?start=24&length=42)

this is not a LIST operation

Reply:
I was under the impression that since the path is .......service/serviceId/validatePolicies 
without individual ID and we validate all the policies it should be a list operation 

what would you recommend

Reply:
a LIST operation is a http GET on a resource collection. Here this API is a POST action so its not a list

Reply:
updated